### PR TITLE
Add documentation for the individual utility functions

### LINKED ISSUE #10: Write better usage documentation
Things to include:
* What do the various functions do?
* What arguments do they take?
* Why would I want to use them?

### PR DESCRIPTION
**Pull request by randycoulman**
Closes #10

### CODE REVIEW

**Review by casto101 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,14 +13,166 @@ npm install --save zeal-redux-utils
 
 ## Usage
 
-```javascript
-import {
-  createActionTypes,
-  createReducer,
-  globalizeSelectors
-} from 'zeal-redux-utils'
+zeal-redux-utils provides a number of utility functions that are useful in [Redux](http://redux.js.org/) applications, especially those that use a [modular or domain-style structure](https://jaysoo.ca/2016/02/28/organizing-redux-application/).
```
Comment (seanculver) [inline README.md:16]: Why is it especially useful for modular or domain-style structures? How does this make it better?

Comment (seanculver) [reply README.md:16]: Possibly answered below =)

**Comment by seanculver** [comment]
This is some really excellent documentation.  👍  :shipit: 

I think it's good enough to deliver. If there is some additional feedback from someone less experienced with redux then those would be great additions to this base at a later time.

File: README.md (hunk 1 above)
Comment (seanculver) [reply README.md:16]: Answered below!

**Review by seanculver — APPROVED** [review]

**Pull request merged**